packages/ui/package.json

```diff
@@ -29,19 +29,18 @@
     "@turbo/gen": "^2.5.4",
     "@types/eslint": "^8.56.5",
     "@types/node": "^20.11.24",
-    "@types/react": "18.3.0",
-    "@types/react-dom": "18.3.1",
+    "@types/react": "^19",
+    "@types/react-dom": "^19",
     "eslint": "^8.57.1",
     "typescript": "5.5.4"
   },
   "dependencies": {
-    "@supabase/auth-ui-react": "0.4.7",
```

Comment:
Brought in React 18.  Newer versions might support React 19, but I didn't check.  It is currently unused so I just removed it.

Reply:
Good call, it is now clear it will not be used.